bin/wasabi.sh

```diff
@@ -302,7 +302,7 @@ package() {
    done)
   (cd modules/ui; \
     mkdir -p target; \
-    for f in app node_modules bower.json Gruntfile.js constants.json karma.conf.js karma-e2e.conf.js package.json test .bowerrc; do \
+    for f in build/constants.json app node_modules bower.json Gruntfile.js default_constants.json karma.conf.js karma-e2e.conf.js package.json test .bowerrc; do \
```

Comment:
I don't think this makes sense?  That is, the "build/constants.json".  This is created by the grunt build (specifically, the config:dist step) and that is done *after* this step in wasabi.sh.  So the build/constants.json doesn't exist, yet, does it?